firmware/library/L1_Drivers/LCD_I2C_Backpack/I2CBackpack.hpp

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include <cstdint>
+
+// Import SJOne libraries to be used for testing
+#include "L2_Drivers/base/i2c_base.hpp"
```

Comment:
This is fine that you are using this for testing, but ti shouldn't be in the commit.